credits.txt

```diff
@@ -258,25 +271,29 @@ contributed to Endless Sky:
   reizbar
   RestingImmortal
   retrue
+  RighthandSon
```

Comment:
This is a dupe of Gods-Righthand.

Reply:
So this is another "name update" one where they changed account name. Thanks, will be fixed momentarily.